proposals/0281-visible-forall.rst

```diff
@@ -778,8 +778,9 @@ and fails on expressions outside of this subset.
 * Variables and constructors (regardless of their namespace) are mapped
   directly, without modification.
 
-  * In the type checking environment, the variable must stand for a type variable,
-    or else it treated as a fresh skolem constant.
+  * In the type checking environment, the variable must stand for a type variable.
+
+    * `#378 Design for Dependent Types <https://github.com/ghc-proposals/ghc-proposals/blob/master/proposals/0378-dependent-type-design.rst#term-variables-in-types>`_ allows term variables in types. However, while it presents how this can be useful with ``foreach``, it's much less clear that it can provide utility with ``forall``. Hence, we save this part of the design of DH for a future proposal.
```

Comment:
But that section *does not even mention `foreach`*!

Reply:
good point, I added it to the referenced section and reworded this paragraph a bit